packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/index.tsx

```diff
@@ -186,6 +186,7 @@ class StepFetchDevfile extends AbstractLoaderStep<Props, State> {
     });
 
     try {
+      SessionStorageService.update(SessionStorageKey.AUTHENTICATION_STATUS, 'started');
```

Comment:
I think this line should be a method in the OAuth service.

Reply:
done